<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendList</class>
 <widget class="QDialog" name="FriendList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>642</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resourse.qrc">
     <normaloff>:/tx/22.png</normaloff>:/tx/22.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>刘泽泽</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_feeling">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>啦啦啦啦啦</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tw_page">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_space">
    <attribute name="title">
     <string>空间</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="page_friend">
    <attribute name="title">
     <string>联系人</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>501</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>319</width>
        <height>499</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="wdg_list" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>340</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_message">
    <attribute name="title">
     <string>消息</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>600</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
